medusa/post_processor.py

```diff
@@ -264,7 +264,9 @@ def _search_files(path, pattern='*', subfolders=None, base_name_only=None, sort=
         path = Path(directory)
         glob = path.rglob(pattern) if subfolders else path.glob(pattern)
 
-        files = [text_type(match) for match in glob]
+        found_items = [text_type(match) for match in glob]
+        # We should only returns files and not folders.
+        files = [f for f in found_items if os.path.isfile(f)]
 
         if sort:
             files = sorted(files, key=os.path.getsize, reverse=True)
```

Comment:
Then this could be a function to be applied to _search_files later:
```python
def sorted_by_size(files, direction='asc'):
     reversed = dict(asc=False, desc=True)
     return sorted(files, key=os.path.getsize, reverse=reversed.get(direction))
```
This would keep _search_files lightweight as a generator, and still give an easy way to get a sorted list later